### PR TITLE
Get Product List

### PR DESCRIPTION
**Pull request by AmrutaDhage20**

### CODE REVIEW

File: config.properties [hunk 1]
```diff
@@ -1,2 +1,2 @@
-//baseURL=https://automationexercise.com/api/productsList
-baseURL=https://automationexercise.com/api/brandsList
+baseURL=https://automationexercise.com/api/productsList
+//baseURL=https://automationexercise.com/api/brandsList
```
Comment (anushreeSonak) [inline config.properties]: Please  keep only productsList URL

File: src/test/java/org/example/GetAllProductList.java [hunk 2]
```diff
@@ -46,24 +46,20 @@ public GetAllProductList(String productId, String productName, String productPri
     public void getLoggerDisplay() {
         PropertyConfigurator.configure("log4j2.properties");
     }
-
     @Test(priority = 1)
     public void validateStatusCode() throws IOException {
         {
-
```
Comment (anushreeSonak) [inline src/test/java/org/example/GetAllProductList.java]: Remove extra space

File: src/test/java/org/example/GetAllProductList.java [hunk 3]
```diff
@@ -46,24 +46,20 @@ public GetAllProductList(String productId, String productName, String productPri
...
             baseURI = ConfigReader.getUrl();
```
Comment (anushreeSonak) [inline src/test/java/org/example/GetAllProductList.java:53]: Please add ConfigReader file

File: src/test/java/org/example/GetAllProductList.java (hunk 2 above)
Comment (anushreeSonak) [inline src/test/java/org/example/GetAllProductList.java]: Remove extra space

File: src/test/java/org/example/GetAllProductList.java [hunk 4]
```diff
@@ -24,24 +24,20 @@ public class GetAllProductList {
...
-
         try {
             url = ConfigReader.getUrl();
         } catch (IOException e) {
             throw new RuntimeException(e);
         }
     }
-
     public GetAllProductList(String productId, String productName, String productPrice, String productBrand) {
-
```
Comment (anushreeSonak) [inline src/test/java/org/example/GetAllProductList.java]: remove extra space

File: src/test/java/org/example/GetAllProductList.java [hunk 5]
```diff
@@ -24,24 +24,20 @@ public class GetAllProductList {
     private String productBrand;
 
     public GetAllProductList() {
-
```
Comment (anushreeSonak) [inline src/test/java/org/example/GetAllProductList.java]: remove extra space

File: src/test/java/org/example/GetAllProductList.java [hunk 6]
```diff
@@ -24,24 +24,20 @@ public class GetAllProductList {
...
     public GetAllProductList(String productId, String productName, String productPrice, String productBrand) {
-
         this.productId = productId;
         this.productName = productName;
         this.productPrice = productPrice;
         this.productBrand = productBrand;
     }
 
     private static Logger logger = Logger.getLogger("Product.class");
-
```
Comment (anushreeSonak) [inline src/test/java/org/example/GetAllProductList.java]: Please make it Final

Comment (AmrutaDhage20) [reply src/test/java/org/example/GetAllProductList.java]: Done

File: src/test/java/org/example/GetAllProductList.java (hunk 5 above)
Comment (AmrutaDhage20) [reply src/test/java/org/example/GetAllProductList.java]: Done

File: src/test/java/org/example/GetAllProductList.java (hunk 4 above)
Comment (AmrutaDhage20) [reply src/test/java/org/example/GetAllProductList.java]: Done

File: src/test/java/org/example/GetAllProductList.java (hunk 2 above)
Comment (AmrutaDhage20) [reply src/test/java/org/example/GetAllProductList.java]: Done

File: src/test/java/org/example/GetAllProductList.java (hunk 3 above)
Comment (AmrutaDhage20) [reply src/test/java/org/example/GetAllProductList.java:53]: Done

File: src/test/java/org/example/GetAllProductList.java (hunk 2 above)
Comment (AmrutaDhage20) [reply src/test/java/org/example/GetAllProductList.java]: Done

File: src/test/java/org/example/GetAllProductList.java [hunk 7]
```diff
@@ -16,31 +17,27 @@
 import static io.restassured.RestAssured.*;
 
 public class GetAllProductList {
-
```
Comment (anushreeSonak) [inline src/test/java/org/example/GetAllProductList.java]: remove extra space

**Review by anushreeSonak — APPROVED** [review]
Approving PR

File: src/test/java/org/example/GetAllProductList.java (hunk 7 above)
Comment (AmrutaDhage20) [reply src/test/java/org/example/GetAllProductList.java]: Done

File: src/test/java/org/example/GetAllProductList.java [hunk 8]
```diff
@@ -16,31 +16,27 @@
...
     private String productName;
     private String productPrice;
     private String productBrand;
 
     public GetAllProductList() {
-
         try {
             url = ConfigReader.getUrl();
         } catch (IOException e) {
             throw new RuntimeException(e);
```
Comment (Vrushalee1234) [inline src/test/java/org/example/GetAllProductList.java]: Add loggers. Instead of throwing exception.

**Review by Vrushalee1234 — CHANGES_REQUESTED** [review]

File: config.properties [hunk 9]
```diff
@@ -1,2 +1 @@
-//baseURL=https://automationexercise.com/api/productsList
-baseURL=https://automationexercise.com/api/brandsList
+baseURL=https://automationexercise.com/api/productsList
```
Comment (amitsahu8) [inline config.properties:1]: Remove .log, target and add .gitignore file

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: config.properties (hunk 9 above)
Comment (AmrutaDhage20) [reply config.properties:1]: Done

File: src/test/java/org/example/GetAllProductList.java (hunk 8 above)
Comment (AmrutaDhage20) [reply src/test/java/org/example/GetAllProductList.java]: Done

**Review by amitsahu8 — APPROVED** [review]

**Review by amitsahu8 — APPROVED** [review]

**Pull request merged**